### DISCUSSION TITLE
glibc-packages for termux-armv7 gcc compile cmake 3.13 Bad system call

### DISCUSSION THREAD

**Discussion by szllzs** [termux-pacman/glibc-packages#40] [Q&A]
![2](https://github.com/termux-pacman/glibc-packages/assets/13085785/f0d9c0e4-2d44-4f8b-8af2-7eda43dbdbac)

I tried the 3.17, 3.13 and 3.27 source code, but all of them produced a "bad system call" error during bootstrap or configure

**Comment by Maxython** [comment]
This discussion is closed, but your issue is moved to #43
